src/main/java/com/pinterest/secor/main/LogFileVerifierMain.java

```diff
@@ -49,11 +49,13 @@ private static CommandLine parseArgs(String[] args) throws ParseException {
                .create("t"));
         options.addOption(OptionBuilder.withLongOpt("start_offset")
                .withDescription("offset identifying the first set of files to check")
+               .hasArg()
```

Comment:
Does this make this arg required argument?  What exception stack trace did you get on the old code?